src/routes/v2/services/slpdb.ts

```diff
@@ -7,7 +7,7 @@ import {
 import axios, { AxiosResponse } from "axios"
 
 export class Slpdb implements ISlpData {
-  async getTokenStats(tokenId: string): Promise<TokenInterface> {
+  public async getTokenStats(tokenId: string): Promise<TokenInterface> {
```

Comment:
Just did some research to verify this: Class methods are public by default, so adding the `public` keyword is redundant. No need to change this if you don't want. Just pointing it out.

Reply:
That's right, I thought this might be more clear. Either way is fine with me though.